README.md

```diff
@@ -1,70 +1,84 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# Project Name
 
-## Available Scripts
+> Check the Financial statements of the biggest US tech companies!
 
-In the project directory, you can run:
+![screenshot](./app_screenshot.png)
 
-### `npm start`
+## Built With
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+- Major languages
+    - JavaScript
+    - HTML & CSS
+- Frameworks
+    - React & Redux
+- Technologies used
+    - VSCode
+    - Git & GitHub
+    - FinancialModelingPrep API [(link here)](https://site.financialmodelingprep.com/developer/docs);
+    - NPM
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+## Live Demo
 
-### `npm test`
+[Live Demo Link](stocks-station.netlify.app)
```

Comment:
**Optional**

- [ ] I get a 404 error when I click on the demo link in the readme. 

![image](https://user-images.githubusercontent.com/51296741/147298407-fe46b33c-66ee-4fed-8ae2-ea9bf5ec2a73.png).
I will suggest you edit the current link to include the https like so:  https://stocks-station.netlify.app/